### PR TITLE
Update quotes.json : Add a quote by Anshuman

### PR DESCRIPTION
**Pull request by AvDevx**
## ✅ New Quote Checklist
- [x] The quote(s) added are in alphabetical order and not ❌ duplicates.
- [x] The quote(s) added are about **learning a foreign language**.
- [x] The sentence attribute does not have inner quotation marks, and is ended with a period.
- [x] Copy pasted the raw JSON `quotes.json` file to [JSONLint](https://jsonlint.com/) and made sure that the modified JSON file is valid.

## ✅ New Language Checklist
- [x] There is ❌ no assigned issue or pull request for this new language.
- [x] ➕ Added a new JSON object in the `server/data/languages.json` file.
- [x] Added the **voice** 🎤 attribute supported by ResponsiveVoice API.
- [x] Added the **flag** 🏳 attribute in the format of the twemoji-awesome CSS syntax.
- [x] Added the **romanization** attribute for appropriate languages.
- [x] ➕ Created the `{language.json}` file inside the `server/data/words` folder containing the list of the 3000 words.
- [x] Removed duplicates and noted 🗒 them in the Pull Request.
- [x] Increment `currentLanguagesCount` in `client/src/store/index.js`
- [x] Add new language in `README.md`

### CODE REVIEW

File: server/data/quotes.json [hunk 1]
```diff
@@ -35,6 +35,10 @@
     "author": "Ann Patchett",
     "sentence": "One must not be shy where language is concerned."
   },
+  {
+    "author": "Anshuman vats",
```
Comment (Kvaibhav01) [inline server/data/quotes.json:39]:
```suggestion
    "author": "Anshuman Vats",
```

**Review by Kvaibhav01 — CHANGES_REQUESTED** [review]
Thanks for the PR @AvDevx, please change as suggested.

**Comment by Kvaibhav01** [comment]
Thanks for updating this!

**Review by AvDevx — COMMENTED** [review]
Is this okay? @Kvaibhav01 
Sorry very new to this don't have much idea how this works.

**Comment by Kvaibhav01** [comment]
You did good @AvDevx, keep it up 👍

**Comment by AvDevx** [comment]
> You did good @AvDevx, keep it up 👍

**Comment by AvDevx** [comment]
Sorry to bother you is there anything else I need to do here or I can move to next issue?

**Comment by Kvaibhav01** [comment]
It's done here @AvDevx, your PR will be accepted and merged soon. Feel free to work on other issues.

**Comment by jayehernandez** [comment]
Thank you for contributing to Letra ☀️, @AvDevx!! Went ahead and resolved the merge conflicts too. Appreciate your contribution to this open source project. 🥳

---

Continue supporting Letra by:
- Leaving a star ⭐️ for this GitHub repository
- Downloading the free [Chrome extension](https://chrome.google.com/webstore/detail/letra/cjodkkjokggcaeacdhjliobekbnnmoio) and leaving a review
- Upvoting the [ProductHunt page](https://www.producthunt.com/posts/letra-2)

**Comment by AvDevx** [comment]
@Kvaibhav01 This is not accepted for hacktoberfest right?

**Pull request merged**